resources.go

```diff
@@ -611,6 +614,8 @@ func Provider() tfbridge.ProviderInfo {
 			"azurerm_shared_image_gallery": {Tok: azureResource(azureCompute, "SharedImageGallery")},
 			"azurerm_shared_image_version": {Tok: azureResource(azureCompute, "SharedImageVersion")},
 			"azurerm_bastion_host":         {Tok: azureResource(azureCompute, "BastionHost")},
+			"azurerm_dedicated_host_group": {Tok: azureResource(azureCompute, "DedicatedHostGroup")},
+			"azurerm_disk_encryption_set":  {Tok: azureResource(azureCompute, "DiskEncryptionSet")},
```

Comment:
Dumb question: I think we used to sort resources within each group alphabetically, but now they are kind of messed up. Do we have a convention about that?

Reply:
There isn't any convention around this - we can do it but there are so many resources that it's hard to keep up